src/ProjectWindow.cpp

```diff
@@ -1942,6 +1942,12 @@ void ProjectWindow::HideEffectsPanel()
    Layout();
 }
 
+bool ProjectWindow::IsEffectsPanelShown()
+{
+   wxWindowUpdateLocker freeze(this);
```

Comment:
What this line is for?

Reply:
I was not sure if it was needed as mEffectsWindows is passed to mContainerWindow's SplitVertically in both ShowEffectsPanel and HideEffectsPanel.

I will removed the lock.